include/GPUInfoProvider.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <lug/Config.hpp>
+#include <lug/Graphics/Renderer.hpp>
+#include <lug/Graphics/Vulkan/API/RTTI/Enum.hpp>
+#include <lug/Graphics/Vulkan/Renderer.hpp>
+
+#if defined(LUG_SYSTEM_WINDOWS)
+    #pragma warning(push)
+    #pragma warning(disable : 4003)
+#endif
+#include <json/json.hpp>
+#if defined(LUG_SYSTEM_WINDOWS)
+    #pragma warning(pop)
+#endif
+
+namespace GPUInfoProvider {
+
```

Comment:
No need to put a new line between the namespace and the prototype of the function